lib/components/side_menu.dart

```diff
@@ -10,37 +11,51 @@ class SideMenu extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(title: Text(title)),
-      drawer: Drawer(
-        child: ListView(
-          padding: EdgeInsets.zero,
-          children: <Widget>[
-            DrawerHeader(
-              child: Text('Home'),
-              decoration: BoxDecoration(
-                color: Colors.blue,
-              ),
-            ),
-            sideMenuListTile(context, 'Home', Home()),
-            sideMenuListTile(context, 'Profile', ProfileScreen()),
-            sideMenuListTile(context, 'Create New Post', NewPostScreen()),
-            sideMenuListTile(context, 'Settings', SettingsScreen()),
-          ],
+    return Drawer(
+        child: Column(children: <Widget>[
+      UserAccountsDrawerHeader(
+        decoration: BoxDecoration(
+          color: Colors.blueGrey[800],
         ),
+        accountName: Text('Tony N.'),
+        accountEmail: Text('tonynguyen@gmail.com'),
```

Comment:
this looks great, just question: do we have ability to pull username from our session yet?

Reply:
not yet, login has to be merged in first